lib/danger/request_sources/github.rb

```diff
@@ -76,7 +76,7 @@ def submit_pull_request_status!(warnings: nil, errors: nil, details_url: nil)
       # use a read-only GitHub account
       if errors.count > 0
         # We need to fail the actual build here
-        abort("danger found #{errors.count} error(s) and doesn't have write access to the PR")
+        abort("\nDanger has failed this build. \nFound #{errors.count} error(s) and I don't have write access to the PR set a PR status.")
```

Comment:
Yeah, the `I` is something I struggled with. The `\n`s I'm sure of. They make it [read much better](https://travis-ci.org/KrauseFx/danger/builds/101248713#L336).

I felt like it was difficult to give the tool a consistent voice, I wanted it to feel somewhat humane, which is I why I keep asking Danger how she'd react. In the end, I opted for anthropomorphising because I figured it would both be useful here, and probably in the future too.